apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -104,7 +106,8 @@ class SectionAssessments extends Component {
 
   state = {
     freeResponseDetailDialogOpen: false,
-    multipleChoiceDetailDialogOpen: false
+    multipleChoiceDetailDialogOpen: false,
+    matchDetailDialogOpen: false
```

Comment:
Is there ever a scenario where these two dialogs are trying to be open at the same time?

Reply:
I don't think so. these open when you click on the see more for the question